requirement.txt

```diff
@@ -0,0 +1,2 @@
+numpy==1.12.0
+scikit-image==0.12.3
```

Comment:
should be merged with utku's old "requirement" file 
-keras
-numpy
-pandas
-tensorflow